README.md

```diff
@@ -159,3 +161,5 @@ The result (at most) the following structure:
          }
 }
 ```
+
+The `options` contains the configuration object passed to `typeset`; this can be useful for passing other data along.
```

Comment:
or for identifying which `typeset()` call is associated with this `callback` call (in case you use the same `callback` function for more than one `typeset()`